backend/fittoring/src/main/java/fittoring/mentoring/infra/SmsRestClientService.java

```diff
@@ -42,4 +44,23 @@ public void sendSms(String to, String text, String subject) {
                 })
                 .body(String.class);
     }
+
+    public void sendSms(Phone toPhone, String text) {
+        smsRestClient.post()
+                .uri(SEND_MESSAGE_ENDPOINT)
+                .header("Authorization", authHeaderGenerator.createAuthorization())
+                .body(Map.of("messages", List.of(new ShortSmsSendClientDto(
+                        toPhone.getNumber(),
```

Comment:
toPhoen은 현재 number만 필요한 것으로 보입니다. 그럼 매개변수로 number만 받는 것은 어떨까요?
그럼 테스트시에 Phone 객체를 준비하지 않아도 되는 장점도 있을 것 같습니다!

```suggestion
    public void sendSms(String toPhoneNumber, String text) {
        smsRestClient.post()
                .uri(SEND_MESSAGE_ENDPOINT)
                .header("Authorization", authHeaderGenerator.createAuthorization())
                .body(Map.of("messages", List.of(new ShortSmsSendClientDto(
                        toPhoneNumber,
```

Reply:
원시 문자열을 매개변수로 받으면 전화번호 형식을 강제할 수 없다는 문제가 있었습니다.  그래서 매개변수로 Phone을 받는 것이 더 안전하다고 생각하여 Phone 객체로 받는 방식으로 변경했습니다.